metrics/core/src/main/java/org/hyperledger/besu/metrics/prometheus/MetricsConfiguration.java

```diff
@@ -207,6 +227,11 @@ public Builder enabled(final boolean enabled) {
       return this;
     }
 
+    public Builder protocol(final String protocol) {
```

Comment:
This isn't set anywhere.  There should be a CLI flag in `BesuCommand.java`.  Probably `--metrics-protocol` (around line 647) accepting the protocol enum and defaulting to prometheus (like we do for `--sync-mode`), then feeding that into the metrics config in the `metricsConfiguration()` method.

Reply:
Ok added.